usr/sbin/truenas-install

```diff
@@ -282,8 +288,8 @@ install_loader()
 	    chroot ${_mnt} grub-install --target=i386-pc /dev/${_disk}
 
 	    echo "Stamping EFI loader on: ${_disk}"
-	    chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}2
-	    chroot ${_mnt} mount -t vfat /dev/${_disk}2 /boot/efi
+	    chroot ${_mnt} mkdosfs -F 32 -s 1 -n EFI /dev/${_disk}p2
```

Comment:
Wont this break for non-nvme?

Reply:
Yes it would, but it actually didn’t work either. For some reason it still doesn’t use the letter p during actual post install tasks. This is where I got stuck. If anyone can implement something better here, I can test it on my system. If not, I can play around with it more tonight.

Reply:
So I found where the logic is coming from..I’m just curious but why is the efi boot logic here and also in the file __main__.py in the truenas_install directory within the truenas-build repo?

Reply:
I was finally able to successfully install to an nvme disk by changing these same lines in the truenas_install/__main__.py file in the truenas-build repo. But you are correct, this logic will need to account for whether it's nvme or non-nvme. I'm not entirely sure where to change it, since it appears it wasn't using this file's install_loader() function to perform the operation.

Reply:
I updated this logic here and here -> https://github.com/truenas/truenas-build/pull/35

I'm going to test it again to make sure my syntax and everything is correct, but this should be it for adding nvme support.